src/components/templates/Playa/PlayaBackground.tsx

```diff
@@ -12,31 +13,29 @@ interface Props {
   nightCycle?: boolean;
 }
 
-export const PlayaBackground: FC<Props> = ({ backgroundImage, nightCycle }) => {
+export const PlayaBackground: React.FC<Props> = ({
+  backgroundImage,
+  nightCycle,
+}) => {
   const [daypartClassName, setDaypartClassName] = useState("");
 
-  useEffect(() => {
-    const updateStats = () => {
-      const pt = dayjs().tz("America/Los_Angeles");
-      const ptHour = pt.hour();
+  useInterval(() => {
+    const pt = dayjs().tz("America/Los_Angeles");
```

Comment:
OOS/future: The timezone should be moved to settings or venue config.

Reply:
Or user config.

Reply:
> Or user config.

That's a good idea. Something like how we do `mirrorVideo`.

Reply:
Exactly!

Reply:
Backlogged in https://github.com/sparkletown/internal-sparkle-issues/issues/14